cellrank/tools/_utils.py

```diff
@@ -1293,11 +1296,19 @@ def _series_from_one_hot_matrix(
         Pandas Series, indicating cluster membership for each sample. The dtype of the categories is `str`
         and samples that belong to no cluster are assigned `NaN`.
     """
-    n_clusters, n_samples = a.shape[1], a.shape[0]
-    if not type(a) == np.ndarray:
+    n_samples, n_clusters = a.shape
+    if not isinstance(a, np.ndarray):
+        raise TypeError(
+            f"Expected `a` to be of type `numpy.ndarray`, found `{type(a).__name__!r}`."
```

Comment:
Note to self: make sure the matrix is an array in `_series_from_one_hot_matrix` as well.